server/server.js

```diff
@@ -100,36 +100,26 @@ function config() {
   var tokens = fs.readFileSync(configPath)
     .toString()
     .split(RegExp(/\s+/));
+  var dataPath;
   for (var i = 0; i < tokens.length; i += 3) {
     var variable = tokens[i];
     var value = tokens[i + 2];
     switch (variable) {
-      case 'bindingPath':
-        bindingPath = value;
-        break;
-      case 'networkPath':
-        networkPath = value;
-        break;
-      case 'expressionPath':
-        expressionPath = value;
+      case 'dataPath':
+        dataPath = value;
         break;
       case 'bigWigToWigPath':
         bigWigToWigPath = value;
         break;
-      case 'uploadPath':
-        uploadPath = value;
-        break;
-      case 'bedPath':
-        bedPath = value;
-        break;
-      case 'mappingPath':
-        mappingPath = value;
-        break;
-      case 'userPath':
-        userPath = value;
-        break;
     }
   }
+  bindingPath = dataPath + 'wiggle/';
+  networkPath = dataPath + 'network/';
+  expressionPath = dataPath + 'expression/';
+  uploadPath = dataPath + 'upload/';
+  bedPath = dataPath + 'bed/';
+  mappingPath = dataPath + 'mapping/';
+  userPath = dataPath + 'user/';
```

Comment:
Ok  then log path could be better

On Tuesday, March 22, 2016, Jiaming Dong <notifications@github.com> wrote:

> In server/server.js
> <https://github.com/ViDA-NYU/genotet/pull/156#discussion_r56998824>:
>
> >      }
> >    }
> > +  bindingPath = dataPath + 'wiggle/';
> > +  networkPath = dataPath + 'network/';
> > +  expressionPath = dataPath + 'expression/';
> > +  uploadPath = dataPath + 'upload/';
> > +  bedPath = dataPath + 'bed/';
> > +  mappingPath = dataPath + 'mapping/';
> > +  userPath = dataPath + 'user/';
>
> Because I am using this path to store the logs, it might be a logPath?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/ViDA-NYU/genotet/pull/156/files/bc04f996734c9c93033743e7338d18aeb48e2d85#r56998824>
>